src/server/certificates/controller.test.js

```diff
@@ -94,6 +95,135 @@ describe('#certificatesController', () => {
   })
 })
 
+describe('API calls for GET/PUT/DELETE', () => {
+  /** @type {Server} */
+  let server
+  let mockGetCertificate, mockPutCertificate, mockDeleteCertificate
+  const apiHeaderKey = config.get('apiAuth')
+  const result = {
```

Comment:
it should be an array of objects, please check the ecert_certificates.json file

Reply:
Resolved. Thanks @alexwbbr